improver_tests/utilities/common_input_handle/test_as_cube.py

```diff
@@ -0,0 +1,53 @@
+# (C) Crown copyright, Met Office. All rights reserved.
+#
+# This file is part of IMPROVER and is released under a BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+from unittest.mock import patch, sentinel
```

Comment:
Yes, these docstrings are a bit superfluous, although I'm a bit reluctant to change our conventions unilaterally in this PR without some discussion/agreement in the team, so as to avoid confusion in other PRs.

Reply:
https://github.com/metoppv/improver/issues/2004